dev.overlayed.Overlayed.yaml

```diff
@@ -0,0 +1,51 @@
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+
+rename-icon: overlayed
+command: overlayed
+finish-args:
+  # NOTE: not sure if this is for both IPC and Websockets transports
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create # allows for discord rpc
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
as mentionned in the previous review comments. it is either x11 alone, or wayland + fallback-x11. Unless there is a specific reason which should be explained.

"I don't know" isn't a proper answer.